### ISSUE TITLE
Scrcpy through Bluetooth

### ISSUE THREAD

**Issue by Diamantaeres** [Genymobile/scrcpy#3060]
Hello,

I would like to set scrcpy over bluetooth from my Nvidia Shield TV to my PC instead of the WiFi - while both the Devices are still connected to the same Wifi - because meanwhile, the WiFi is going to be used by another service that doesn't offer a Bluetooth alternative, so I want to reduce as much as possible the load on the Wifi to make the other service perform as good as possible.

I asked in a previous post but I'm still not clear with how can I do that.

I had this answer (pull 1303):
"Scrcpy does nothing special for bluetooth: if you connect your computer to the device bluetooth network, it should work as usual."

Now, I don't know what's meant by "Bluetooth Network", so I tried to ask around and they told me of "IP over bluetooth" - that I don't know as well - and Bluetooth Tethering, which I know.

Now, I would like to understand, are these 3 things the same thing ("Bluetooth Tethering", "IP over Bluetooth" and "bluetooth network")?

If yes: which of the two devices should be the source? Should Shield connect to PC tethering or should PC connect to Shield tethering? (or it doesn't make the difference and works either way?)

Also, Shield is Android TV based and doesn't have a native Tethering option.
Trying the other way around by opening the Bluetooth Tethering from my PC, it seems that Shield cannot connect to it natively, it has only a normal pair option, which I am also wondering if it can be sufficient for the scrcpy connection.

Do you know if there is some way, maybe if you are aware of an app that may do the job, that allows me to get the needed option working on Android TV without rooting my device? 

And last question: How can I be 100% sure that the connection is passing on Bluetooth and not on Wifi while the two devices are connected to the wifi network for their individual connectivity?

I apologize if my questions may result too basic, silly or out of topic and take this opportunity to thank you in advance for your time and this amazing project, I really love it.

**Comment by Diamantaeres** [comment]
Hello,

I managed to do it, thanks :)

**Comment by rom1v** [comment]
> I managed to do it, thanks :)

Maybe you could explain what you did for future users :)

**Comment by Diamantaeres** [comment]
Hello,

yes I can, just didn't do it because it performs so badly that may not worth it:
my first test was 1 frame every 45 seconds, with your suggestion of -b1M went to 1 every around 20 seconds XD

So, I sideloaded "SecureTether BT" to my Android TV which enables the tethering that is not natively on the device.

This app is great because it also has a step by step guide very precise to configure the connection, it's very detailed, asking you your Device, OS, version and has screenshots on each steps and some of them have selective questions to go ahead with different paths.

Once the two devices are connected, you normally run SCRCPY.

To make sure it was running on Bluetooth and not Wifi, I turned of the PC Wifi.

As I mentioned before, the performance are very very low, so personally I don't think I will use this solution (will keep the app installed anyway).

Regards
